concepts/58 jQuery Components/05 Add DevExtreme to a jQuery Application/05 CDN Files.md

```diff
@@ -1,18 +1,24 @@
 
     <!--HTML-->
-    <script type="text/javascript" src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.4.1.min.js"></script>
+    <head>
+        <!-- ... -->
+        <script type="text/javascript" src="https://ajax.aspnetcdn.com/ajax/jquery/jquery-3.4.1.min.js"></script>
 
-    <!-- DevExtreme themes -->
-    <link rel="stylesheet" href="https://cdn3.devexpress.com/jslib/minor_20_1/css/dx.common.css">
-    <link rel="stylesheet" href="https://cdn3.devexpress.com/jslib/minor_20_1/css/dx.light.css">
+        <!-- DevExtreme themes -->
+        <link rel="stylesheet" href="https://cdn3.devexpress.com/jslib/minor_20_1/css/dx.common.css">
+        <link rel="stylesheet" href="https://cdn3.devexpress.com/jslib/minor_20_1/css/dx.light.css">
 
-    <!-- DevExtreme library -->
-    <script type="text/javascript" src="https://cdn3.devexpress.com/jslib/minor_20_1/js/dx.all.js"></script>
-    <!-- <script type="text/javascript" src="https://cdn3.devexpress.com/jslib/minor_20_1/js/dx.web.js"></script> -->
-    <!-- <script type="text/javascript" src="https://cdn3.devexpress.com/jslib/minor_20_1/js/dx.viz.js"></script> -->
-    <!-- <script type="text/javascript" src="https://cdn3.devexpress.com/jslib/minor_20_1/js/dx.viz-web.js"></script> -->
+        <!-- DevExtreme library -->
+        <script type="text/javascript" src="https://cdn3.devexpress.com/jslib/minor_20_1/js/dx.all.js"></script>
+        <!-- <script type="text/javascript" src="https://cdn3.devexpress.com/jslib/minor_20_1/js/dx.web.js"></script> -->
+        <!-- <script type="text/javascript" src="https://cdn3.devexpress.com/jslib/minor_20_1/js/dx.viz.js"></script> -->
+        <!-- <script type="text/javascript" src="https://cdn3.devexpress.com/jslib/minor_20_1/js/dx.viz-web.js"></script> -->
+    </head>
+    <body class="dx-viewport">
+        <!-- ... -->
+    </body>
 
-The stylesheets in the code above apply the Generic Light theme. See [Predefined Themes](/concepts/60%20Themes%20and%20Styles/05%20Predefined%20Themes/00%20Predefined%20Themes.md '/Documentation/Guide/Themes_and_Styles/Predefined_Themes/') for information about the other available themes.
+The stylesheets in the code above apply the Generic Light theme. The `dx-viewport` class ensures that theme colors and typography settings are applied to all elements on the page, not only to DevExtreme widgets. See [Predefined Themes](/concepts/60%20Themes%20and%20Styles/05%20Predefined%20Themes/00%20Predefined%20Themes.md '/Documentation/Guide/Themes_and_Styles/Predefined_Themes/') for information about other available themes.
```

Comment:
Can we change the first two sentences to:
Stylesheets in the code above apply a Generic Light theme. The `dx-viewport` class ensures that theme colors and typography settings are applied to all page elements (and not only to DevExtreme widgets).